ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__acm-ocp4.15-lp-interop.yaml

```diff
@@ -0,0 +1,94 @@
+base_images:
+  acmqe-grc-test:
+    name: "2.8"
+    namespace: acm-qe
+    tag: acmqe-grc-test
+  application-ui-test:
+    name: "2.8"
+    namespace: acm-qe
+    tag: application-ui-test
+  clc-ui-e2e:
+    name: "2.8"
+    namespace: acm-qe
+    tag: clc-ui-e2e
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  fetch-managed-clusters:
+    name: autotest
+    namespace: acm-qe
+    tag: fetch-managed-clusters
+  observability-core-automation:
+    name: "2.8"
+    namespace: acm-qe
+    tag: observability-core-automation
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: acm-interop-aws
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      CONTROL_PLANE_INSTANCE_TYPE: m6a.2xlarge
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-conf*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
+          {"step": "ipi-install*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
+          {"step": "ipi-deprovision*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
```

Comment:
If you wanted to slim this config down, I see you ignore several "gather" steps. Instead of doing `"gather-*"` here you could do `"*gather*"` and remove lines 62 and 64. Not a big deal if you want to keep this.

Reply:
Good point I'll update this

Reply:
done